firefox_puppeteer/ui/windows.py

```diff
@@ -324,20 +324,25 @@ def open_window(self, callback=None, expected_window_class=None):
         return window
 
     def send_shortcut(self, command_key, **kwargs):
-        """Sends a keyboard shortcut to the window
+        """Sends a keyboard shortcut to the window.
 
-        :param command_key: The key (usually a letter) to be pressed
+        :param command_key: The key (usually a letter) to be pressed.
 
-        :param accel: Optional, If `True` the `Accel` modifier key is pressed. This key
-         differs between OS X (`Meta`) and Linux/Windows (`Ctrl`). Defaults to `False`
+        :param accel: Optional, If `True`, the `Accel` modifier key is pressed.
+                      This key differs between OS X (`Meta`) and
+                      Linux/Windows (`Ctrl`). Defaults to `False`.
```

Comment:
The additional punctuation in the first line made it longer than Travis will accept.